examples/00-fluent/turbo_machinery.py

```diff
@@ -0,0 +1,815 @@
+# Describe turbo surfaces
+# ~~~~~~~~~~~~~~~~~~~~~~~
+# Define turbo-specific iso-surfaces.
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_014.png
+#   :width: 500pt
+#   :align: center
+
+solver_session.workflow.TaskObject["Define Turbo Surfaces"].Arguments.set_state(
+    {
+        "IsoSurfaceNumList": ["surface 3", "surface 2", "surface 1"],
+        "NewIsoSurfaceNameList": ["twf_span_75", "twf_span_50", "twf_span_25"],
+        "NewIsoSurfaceValueList": ["0.75", "0.5", "0.25"],
+        "OldIsoSurfaceNameList": ["twf_span_3", "twf_span_2", "twf_span_1"],
+        "OldIsoSurfaceValueList": ["0.75", "0.5", "0.25"],
+    }
+)
+solver_session.workflow.TaskObject["Define Turbo Surfaces"].Execute()
+
+
+###############################################################################
+# Create report definitions and monitors
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Finalize the workflow and to create postprocessing objects such as contour plots
+# on the specified turbo-surfaces, as well as turbo-specific report definitions and monitors
+
+solver_session.workflow.TaskObject["Create Report Definitions & Monitors"].Execute()
+
+
+###############################################################################
+# Initialize flow field
+# ~~~~~~~~~~~~~~~~~~~~~
+# Initialize the flow field using hybrid initialization.
+
+solver_session.tui.solve.initialize.hyb_initialization()
+
+
+###############################################################################
+# Save case file
+# ~~~~~~~~~~~~~~
+# Save the case file as (``turbo_workflow.cas.h5``).
+
+solver_session.tui.file.write_case("turbo_workflow.cas.h5")
+
+###############################################################################
+# Solve for 100 iterations
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Solve for 100 iterations.
+
+# - Residuals
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_015.png
+#   :width: 500pt
+#   :align: center
+
+# - Isentropic Efficiency of the Compressor
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_016.png
+#   :width: 500pt
+#   :align: center
+
+# - Pressure Ratio
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_017.png
+#   :width: 500pt
+#   :align: center
+
+#  - Total Mass Flow Rate at the Outlet
+
+###############################################################################
+# .. image:: /_static/turbo_machinery_018.png
+#   :width: 500pt
+#   :align: center
+
+solver_session.tui.solve.iterate(100)
+
+
+###############################################################################
+# Save data file
+# ~~~~~~~~~~~~~~
+# Save the data file as (``turbo_workflow.dat.h5``).
+
+solver_session.tui.file.write_data("turbo_workflow.dat.h5")
```

Comment:
Currently fails on v241 here, with the error message:
`Error: Mesh has changed, cannot write the data file without writing the case file`

This should fix it:
```suggestion
solver_session.tui.file.write_case_data("turbo_workflow.cat.h5")
```

Reply:
Changes added.

https://github.com/ansys/pyfluent/blob/1438197148adef64801dbbb1f936653a872bb725/examples/00-fluent/turbo_machinery.py#L802-L807